business-registry-dashboard/app/stores/affiliations.ts

```diff
@@ -42,10 +42,15 @@ export const useAffiliationsStore = defineStore('brd-affiliations-store', () =>
   }
 
   async function removeBusiness (payload: RemoveBusinessPayload) {
-    // If the business is a new registration then remove the business filing from legal-db
-    if (payload.business.corpType.code === CorpTypes.INCORPORATION_APPLICATION) {
+    // If the business is a new IA, amalgamation, continuation in, or registration then remove the business filing from legal-db
```

Comment:
We want to allow deletion of Continuation In filing only when it's in DRAFT status. After that there's a review record linked to it and we haven't talked about what to do with it, so for now it's not deleteable.

Reply:
Thanks for catching this. I removed this and added a TODO item to add the continuation in later.

Reply:
Fine by me. Resolving.